WcaOnRails/app/helpers/results_helper.rb

```diff
@@ -33,4 +33,19 @@ def link_to_ranking(event_id, type, &block)
     url = "/results/events.php?eventId=#{event_id}&#{type}=true"
     link_to url, class: "plain", &block
   end
+
+  def pb_type_class_for_result(regionalRecord,pbMarker)
+    if pbMarker
+      'pb'
```

Comment:
I think Travis is complaining about this `'pb'` left alone here. Maybe you could change this line (and the ones below) to be `class = 'pb'` (or the respective record.)